src/devices/bus/sg1000_exp/sp400.cpp

```diff
@@ -0,0 +1,201 @@
+/*
```

Comment:
this entire file is using indentation that diverges from the rest of the codebase

Reply:
Thanks, I will rebase. As for the identation, l'll tabify it (I am an emacs user, hence the spaces). Is there any "mame-fy" tool?